plugins/healthchecks/healthchecks.c

```diff
@@ -301,8 +301,22 @@ parse_configs(const char *fname)
   char buf[2 * 1024];
   HCFileInfo *head_finfo = NULL, *finfo = NULL, *prev_finfo = NULL;
 
-  if (NULL == (fd = fopen(fname, "r")))
+  if (!fname) {
     return NULL;
+  }
+
+  if ('/' == *fname) {
+    fd = fopen(fname, "r");
+  } else {
+    char filename[PATH_MAX + 1];
+
+    snprintf(filename, sizeof(filename), "%s/%s", fname, TSConfigDirGet());
+    fd = fopen(filename, "r");
```

Comment:
So, these files are the actual content files, and they really should not be based in the ATS config area (or anything near ATS run-time). Best practices says that you likely want to use an absolute path in these config files, I can add that to the docs.

Also, all this changes if/when we replace this junk with a file serving remap based plugin :).